docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -119,26 +119,186 @@ that catalog name instead of ``sqlserver`` in the above examples.
 Type mapping
 ------------
 
-Trino supports the following SQL Server data types:
-
-==================================  ===============================
-SQL Server Type                     Trino Type
-==================================  ===============================
-``bigint``                          ``bigint``
-``tinyint``                         ``smallint``
-``smallint``                        ``smallint``
-``int``                             ``integer``
-``float``                           ``double``
-``char(n)``                         ``char(n)``
-``varchar(n)``                      ``varchar(n)``
-``date``                            ``date``
-``datetime2(n)``                    ``timestamp(n)``
-``datetimeoffset(n)``               ``timestamp(n) with time zone``
-==================================  ===============================
+Because Trino and SQL Server each support types that the other does not, this
+connector modifies some types when reading or writing data.
+
+SQL Server to Trino read type mapping
```

Comment:
Is "read type mapping" the best term to use? That would imply that this is only relevant when performing read operations (https://trino.io/docs/current/language/sql-support.html#read-operations), but I'd suggest it's better to just call this "SQL Server type to Trino type mapping", and the following section "Trino type to SQL Server type mapping" without read/write being part of it

Reply:
Good suggestion!